modules/nf-core/somatosim/somatosim/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM nfcore/base:2.1
+LABEL \
+    author="Maxime Garcia" \
+    description="snpEff image for nf-core pipelines" \
+    maintainer="maxime.garcia@scilifelab.se"
```

Comment:
I think the `LABEL` is supposed to be:

```
LABEL org.opencontainers.image.authors="SvenDowideit@home.org.au"
```

https://docs.docker.com/engine/reference/builder/#maintainer-deprecated

And maybe not @maxulysse 🤔